### ISSUE TITLE
Wrong event times

### ISSUE THREAD

**Issue by sunkup** [bitfireAT/icsx5#419]
Thunderbird (and apparently other calendar software) imports the event in the following calendar as happening at 16:30, while ICSx5 imports it at 01:00 am (same day). 

```
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//wp-events-plugin.com//6.5.2//EN
TZID:Europe/Berlin
X-WR-TIMEZONE:Europe/Berlin
BEGIN:VEVENT
UID:583@dokus.uni-stuttgart.de
DTSTART;TZID=Europe/Berlin:20240925T163000
DTEND;TZID=Europe/Berlin:20240925T180000
DTSTAMP:20240910T154945Z
URL:https://dokus.uni-stuttgart.de/events/dokus-sitzung-2024-09-24/
SUMMARY:DoKUS-Sitzung
DESCRIPTION:Die DoKUS-Sitzung
END:VEVENT
BEGIN:VTIMEZONE
TZID:Europe/Berlin
X-LIC-LOCATION:Europe/Berlin
BEGIN:DAYLIGHT
DTSTART:20240331T030000
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
END:DAYLIGHT
BEGIN:STANDARD
DTSTART:20241027T020000
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20250330T030000
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
END:DAYLIGHT
BEGIN:STANDARD
DTSTART:20251026T020000
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
END:STANDARD
BEGIN:DAYLIGHT
DTSTART:20260329T030000
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
END:DAYLIGHT
END:VTIMEZONE
END:VCALENDAR
```

Discussed in https://github.com/bitfireAT/icsx5/discussions/188

**Comment by ArnyminerZ** [comment]
I _think_, as far as I can remember, that we couldn't do much, it had to be fixed by WEM

**Comment by sunkup** [comment]
Correct. Unfortunately we are on the final release of ical4j and won't be able to update to 4.x.x without a lot of work - not to mention DAVx5 would have to get updated at the same time, unless we create a separate branch in ical4android.

**Comment by sunkup** [comment]
> We can try on 4.0.4 using the test I made for https://github.com/ical4j/ical4j/issues/651, I think it shouldn't be too much.

Your test does not fail anymore. Not sure why, but maybe because we are in daylight saving time now? I tried with another example from the discussion and got a parse exception https://github.com/ical4j/ical4j/issues/651#issuecomment-2345738053 .

**Comment by sunkup** [comment]
Will probably not be able to solve this within the next version.

**Comment by q-wertz** [comment]
I don't really know why, but the times are now correct in my calendar...
Cannot really see a change in [Wordpress Events Manager](https://de.wordpress.org/plugins/events-manager/) and I also think there was no new ICSx5 version.
Just noticed yesterday, that the events have the correct time...

Some time ago I changed this setting in Events Manager to "No" but didn't see any improvement. Maybe it just took some time for sync?
![image](https://github.com/user-attachments/assets/0bc2aa82-ef16-40d6-ac46-24df08f3c37d)

On the other hand side the current `ics` 
```
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//wp-events-plugin.com//6.6.1//EN
TZID:Europe/Berlin
X-WR-TIMEZONE:Europe/Berlin
BEGIN:VEVENT
UID:776@dokus.uni-stuttgart.de
DTSTART;TZID=Europe/Berlin:20241022T163000
DTEND;TZID=Europe/Berlin:20241022T180000
DTSTAMP:20241008T183142Z
URL:https://dokus.uni-stuttgart.de/events/dokus-sitzung-2024-10-29/
SUMMARY:DoKUS-Sitzung
CATEGORIES:DoKUS-Sitzung
END:VEVENT
[...]
```
looks the same way to me as reported [earlier](https://github.com/bitfireAT/icsx5/discussions/188#discussioncomment-10604080)
